app/scripts/models/ThreeViewer.js

```diff
@@ -0,0 +1,306 @@
+define(['backbone', 'Viewport', 'staticHelpers', 'GeometryExport'], function (Backbone, Viewport, staticHelpers, GeometryExport) {
+
+    var colors = {
+        selected: 0x00FFFF,
+        notSelected: 0x999999,
+        notSelectedLine: 0x000000
+    };
+
+    function colorSelected(threeGeom, selected) {
+
+        var partMat,
+            meshMat,
+            lineMat;
+
```

Comment:
These could get hoisted out of the function and made constant.  http://adsk-oss.myjetbrains.com/youtrack/issue/REACH-193